scripts/upgrade/rtorrent.sh

```diff
@@ -48,6 +72,9 @@ else
     rtorrent_apt
     echo_progress_done
 fi
+echo_progress_start "Updating systemd serivce "
+_systemd
```

Comment:
We'll need to call systemctl daemon-reload at some point to ensure the unit file changes are loaded.